src/main/scala/mesosphere/marathon/core/deployment/impl/TaskReplaceActor.scala

```diff
@@ -80,13 +80,27 @@ class TaskReplaceActor(
     instancesByIncarnation._1.sortBy(_.appTask.status.stagedAt) ++ instancesByIncarnation._2
   }
 
+  // Return the number of non running old instances killed
+  def killNonRunningOldInstances(oldInstances: Seq[Instance]): Integer = {
```

Comment:
OPT: This method could be just named killNonRunningInstances and parameter could be changed to.
Doing this would all us to reuse this method if needed; + it's clearer imo as nothing in this method does filtering between old/new.

Reply:
I agree on the fact that nothing in this method does filtering between old and new instances, and that the name of this method could be shorter.
However, this name is a reminder that this method should be used on old instances only (I doubt we want to kill new instances in the TaskReplaceActor) + it is consistent with the name and action of the killNextOldInstance method, which is called inside killNonRunningOldInstances.